common/static/common/js/discussion/discussion_router.js

```diff
@@ -0,0 +1,167 @@
+/* globals DiscussionThreadListView, DiscussionThreadView, DiscussionUtil, NewPostView */
```

Comment:
Ignore, I'm seeing this was in the original javascript - I thought Coffeescript was adding this